raiden/network/rpc/client.py

```diff
@@ -217,12 +224,10 @@ def gaslimit(self, location='latest') -> int:
         gas_limit = self.web3.eth.getBlock(location)['gasLimit']
         return gas_limit * 8 // 10
 
-    @cachedmethod(attrgetter('_gasprice_cache'))
```

Comment:
You should also remove `self._gasprice_cache` then

Reply:
Good catch